### DISCUSSION TITLE
Unable to scroll while selecting files in dde-file-manager

### DISCUSSION THREAD

**Discussion by NjlsShade** [linuxdeepin/developer-center#3397] [Features Request & Ideas | 特性请求 & 头脑风暴]
### SUMMARY | 问题概要

When dragging to select multiple files in deepin file manager I am unable to scroll up and down (using the scroll wheel) in the folder to quickly select in bulk, instead I am forced to use the slower method of hovering near the top and bottom of the file manager windows to seek up and down.

### STEPS TO REPRODUCE | 复现步骤

1. In a folder with enough content to have a scrollbar, click and hold to start box-selecting items in the file manager
![Screenshot_20220520220347](https://user-images.githubusercontent.com/15788214/169631437-23cfc614-84ab-40a5-9565-bb1d535b835c.png)

2. Now, while holding down left click try and scroll to select more items off-screen

### OBSERVED RESULT | 观察到的结果

The page does not scroll

### EXPECTED RESULT | 期望的结果

The page should scroll as it does in most other areas such as the web browser:
![Record_select-area_20220520225024](https://user-images.githubusercontent.com/15788214/169631596-9ffa090f-984c-42a5-b745-c15d4203326c.gif)

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

Using the latest deepin 20.5 freshly installed and updated.

**Comment by BLumia** [comment]
Can be confirmed under both deepin v20.5 with `dde-file-manager 5.5.10-1` and the current v20.6 beta `dde-file-manager 5.6.3-1`. As a workaround, dragging the mouse cursor up or down should work.

**Comment by babyfengfjx** [comment]
This is a great feature that is not supported in the current version, perhaps we can consider it later, thank you very much for your feedback .

**Comment by wootifer** [comment]
Version 5.6.3 seems to already support this feature.
Using the left mouse button + Shift/Ctrl to multi-select files also seems to be a good option.

**Comment by BLumia** [comment]
> Version 5.6.3 seems to already support this feature.

I did try on the final released version of dde-file-manager in deepin v20.6 (dde-file-manager 5.6.3-1) and it doesn't support such a feature :(

**Comment by meiyixiang** [comment]
Hello, I have submitted this requirement to the clerical management team
